src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,102 @@
  */
 
 public class Plane {
+    private final String name;
+    private final int seats;
+    private final int wheels;
+    private final int wings;
+    private final int engines;
+    private final int windows;
```

Comment:
what is the purpose of making these fields final here?

Reply:
I was thinking these fields are more protected if make them final, and little bit more difficult to change them so I tried to put final and to build Builder for this type of field. Anyway, I changed them to private.